### PR TITLE
MGMT-8564 - Remove kube-api project path dependency

### PR DESCRIPTION
**Pull request by eliorerz**
/cc @osherdp

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-ocm-2.3.yaml [hunk 1]
```diff
@@ -98,10 +98,9 @@ tests:
       ASSISTED_CONFIG: |
         SERVICE_BASE_REF=ocm-2.3
       POST_INSTALL_COMMANDS: |
-        export TEST=./discovery-infra/tests/test_kube_api.py
         export TEST_FUNC=test_kube_api_ipv4
         export KUBECONFIG=$(find ${KUBECONFIG} -type f)
-        make deploy_assisted_operator test_parallel
+        make deploy_assisted_operator test_kube_api_parallel
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-ocm-2.3.yaml:103]: how? you didn't backport the test-infra PR
same for 2.4

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-ocm-2.3.yaml (hunk 1 above)
Comment (eliorerz) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-ocm-2.3.yaml]: Right. Removed 2.3, v1, and 2.4

**Comment by osherdp** [comment]
/hold
(to see rehearse results...)

**Pull request merged**